pages/instructorAssessmentQuestionStatistics/instructorAssessmentQuestionStatistics.js

```diff
@@ -29,20 +30,27 @@ router.get('/', function (req, res, next) {
   setFilenames(res.locals);
   async.series(
     [
-      function (callback) {
-        var params = { assessment_id: res.locals.assessment.id };
-        sqldb.queryOneRow(sql.assessment_stats_last_updated, params, function (err, result) {
-          if (ERR(err, callback)) return;
-          res.locals.stats_last_updated = result.rows[0].stats_last_updated;
-          callback(null);
+      async () => {
+        // make sure statistics are up to date
```

Comment:
Is it actually necessary to re-compute the statistics for this page? AFAICT it only shows *question* statistics, not stats for the assessment as a whole.

Reply:
That's true, but it uses the student count from the overall assessment statistics to determine whether to show the question statistics at all. I'm making the minimal possible changes to the question statistics pages in this PR (deferring to #6743) so I'm just preserving the use of `assessment.score_stat_number` as a flag.